   ***********************header***********************   
 
时间：2018-05-18 11:45:05,456 
线程ID:[1] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.Net.WebException: 操作超时
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 ZXClient.util.HttpUtil.RequestStream(String POSTURL, String PostData) 位置 D:\workspace\zx\ZXClient\ZXClient\util\HttpUtil.cs:行号 119

 
时间：2018-05-18 11:45:05,518 
线程ID:[1] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 ZXClient.util.HttpUtil.RequestData(String POSTURL, String PostData) 位置 D:\workspace\zx\ZXClient\ZXClient\util\HttpUtil.cs:行号 21

 
时间：2018-05-18 11:45:15,781 
线程ID:[1] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.Net.WebException: 操作超时
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 ZXClient.util.HttpUtil.RequestStream(String POSTURL, String PostData) 位置 D:\workspace\zx\ZXClient\ZXClient\util\HttpUtil.cs:行号 119

 
时间：2018-05-18 11:45:15,816 
线程ID:[1] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 ZXClient.util.HttpUtil.RequestData(String POSTURL, String PostData) 位置 D:\workspace\zx\ZXClient\ZXClient\util\HttpUtil.cs:行号 21

 
时间：2018-05-18 11:45:39,535 
线程ID:[1] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.Net.WebException: 操作超时
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 ZXClient.util.HttpUtil.RequestStream(String POSTURL, String PostData) 位置 D:\workspace\zx\ZXClient\ZXClient\util\HttpUtil.cs:行号 119

 
时间：2018-05-18 11:45:39,559 
线程ID:[1] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 ZXClient.util.HttpUtil.RequestData(String POSTURL, String PostData) 位置 D:\workspace\zx\ZXClient\ZXClient\util\HttpUtil.cs:行号 21
   ***********************footer***********************/r/n    ***********************header***********************      ***********************footer***********************/r/n    ***********************header***********************   
 
时间：2018-05-18 13:13:53,134 
线程ID:[7] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 ZXClient.LoginWindow.AutoLogin() 位置 D:\workspace\zx\ZXClient\ZXClient\LoginWindow.cs:行号 30
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()
   ***********************header***********************   
 
时间：2018-05-18 13:18:02,529 
线程ID:[8] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.Net.Sockets.SocketException: 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 size, SocketFlags socketFlags)
   在 ZXClient.WorkForm.USBSendData(Object response, String op, Boolean isPJQ, UdpClient client, IPEndPoint ipendpoint) 位置 D:\workspace\zx\ZXClient\ZXClient\WorkForm.cs:行号 602

 
时间：2018-05-18 13:18:14,664 
线程ID:[7] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.InvalidOperationException: 在创建窗口句柄之前，不能在控件上调用 Invoke 或 BeginInvoke。
   在 System.Windows.Forms.Control.MarshaledInvoke(Control caller, Delegate method, Object[] args, Boolean synchronous)
   在 System.Windows.Forms.Control.Invoke(Delegate method, Object[] args)
   在 ZXClient.LoginWindow.AutoLogin() 位置 D:\workspace\zx\ZXClient\ZXClient\LoginWindow.cs:行号 64
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()
   ***********************header***********************   
 
时间：2018-05-18 13:22:08,249 
线程ID:[7] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.Net.Sockets.SocketException: 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 size, SocketFlags socketFlags)
   在 ZXClient.WorkForm.USBSendData(Object response, String op, Boolean isPJQ, UdpClient client, IPEndPoint ipendpoint) 位置 D:\workspace\zx\ZXClient\ZXClient\WorkForm.cs:行号 602

 
时间：2018-05-18 13:22:38,324 
线程ID:[6] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.InvalidOperationException: 在创建窗口句柄之前，不能在控件上调用 Invoke 或 BeginInvoke。
   在 System.Windows.Forms.Control.MarshaledInvoke(Control caller, Delegate method, Object[] args, Boolean synchronous)
   在 System.Windows.Forms.Control.Invoke(Delegate method, Object[] args)
   在 ZXClient.LoginWindow.AutoLogin() 位置 D:\workspace\zx\ZXClient\ZXClient\LoginWindow.cs:行号 64
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()
   ***********************header***********************   
 
时间：2018-05-18 13:23:16,911 
线程ID:[8] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.Net.Sockets.SocketException: 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 size, SocketFlags socketFlags)
   在 ZXClient.WorkForm.USBSendData(Object response, String op, Boolean isPJQ, UdpClient client, IPEndPoint ipendpoint) 位置 D:\workspace\zx\ZXClient\ZXClient\WorkForm.cs:行号 602
   ***********************header***********************   
 
时间：2018-05-18 13:27:36,953 
线程ID:[8] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.Net.Sockets.SocketException: 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 size, SocketFlags socketFlags)
   在 ZXClient.WorkForm.USBSendData(Object response, String op, Boolean isPJQ, UdpClient client, IPEndPoint ipendpoint) 位置 D:\workspace\zx\ZXClient\ZXClient\WorkForm.cs:行号 602
   ***********************header***********************   
 
时间：2018-05-18 13:31:38,982 
线程ID:[8] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.Net.Sockets.SocketException: 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 size, SocketFlags socketFlags)
   在 ZXClient.WorkForm.USBSendData(Object response, String op, Boolean isPJQ, UdpClient client, IPEndPoint ipendpoint) 位置 D:\workspace\zx\ZXClient\ZXClient\WorkForm.cs:行号 602
   ***********************header***********************   
 
时间：2018-05-18 13:33:03,456 
线程ID:[8] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.Net.Sockets.SocketException: 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 size, SocketFlags socketFlags)
   在 ZXClient.WorkForm.USBSendData(Object response, String op, Boolean isPJQ, UdpClient client, IPEndPoint ipendpoint) 位置 D:\workspace\zx\ZXClient\ZXClient\WorkForm.cs:行号 602
   ***********************header***********************   
 
时间：2018-05-18 13:33:43,712 
线程ID:[9] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.Net.Sockets.SocketException: 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 size, SocketFlags socketFlags)
   在 ZXClient.WorkForm.USBSendData(Object response, String op, Boolean isPJQ, UdpClient client, IPEndPoint ipendpoint) 位置 D:\workspace\zx\ZXClient\ZXClient\WorkForm.cs:行号 602
   ***********************header***********************   
 
时间：2018-05-18 13:37:04,433 
线程ID:[8] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.Net.Sockets.SocketException: 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 size, SocketFlags socketFlags)
   在 ZXClient.WorkForm.USBSendData(Object response, String op, Boolean isPJQ, UdpClient client, IPEndPoint ipendpoint) 位置 D:\workspace\zx\ZXClient\ZXClient\WorkForm.cs:行号 602

 
时间：2018-05-18 13:38:01,104 
线程ID:[7] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.InvalidOperationException: 在创建窗口句柄之前，不能在控件上调用 Invoke 或 BeginInvoke。
   在 System.Windows.Forms.Control.MarshaledInvoke(Control caller, Delegate method, Object[] args, Boolean synchronous)
   在 System.Windows.Forms.Control.Invoke(Delegate method, Object[] args)
   在 ZXClient.LoginWindow.AutoLogin() 位置 D:\workspace\zx\ZXClient\ZXClient\LoginWindow.cs:行号 64
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()
   ***********************header***********************   
 
时间：2018-05-18 13:40:21,677 
线程ID:[8] 
日志级别：  ERROR 
出错类：ERR property: [(null)] - 
错误描述：Error
 
System.Net.Sockets.SocketException: 通常每个套接字地址(协议/网络地址/端口)只允许使用一次。
   在 System.Net.Sockets.Socket.DoBind(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Bind(EndPoint localEP)
   在 ZXClient.WorkForm.ConnDevice() 位置 D:\workspace\zx\ZXClient\ZXClient\WorkForm.cs:行号 219
